packages/opencensus-core/test/test-tracer.ts

```diff
@@ -52,17 +52,13 @@ describe('Tracer', () => {
 
   /** Should get/set the current RootSpan from tracer instance */
   describe('get/set currentRootSpan()', () => {
-    let tracer, rootSpan;
-    before(() => {
-      tracer = new Tracer();
-      rootSpan = new RootSpan(tracer);
-      tracer.currentRootSpan = rootSpan;
-    });
+    const tracer = new Tracer().start(defaultConfig);
```

Comment:
This change looks unrelated to the Node 6 change. Was this intended?

Reply:
Some unit tests were failing. Related to context issues.